core/src/test/java/io/substrait/type/proto/ExtensionRoundtripTest.java

```diff
@@ -312,7 +312,7 @@ void scalarSubquery() {
                   Stream.of(
                           Expression.ScalarSubquery.builder()
                               .input(relWithEnhancement)
-                              .type(TypeCreator.REQUIRED.struct(TypeCreator.REQUIRED.I64))
+                              .type(TypeCreator.REQUIRED.I64)
```

Comment:
not 100% sure about this, is it actually meant to return a struct type? given it's scalar that seems a bit weird